### PR TITLE
Config map change

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

File: pkg/protoform/installer.go [hunk 1]
```diff
@@ -618,6 +662,7 @@ func (i *Installer) generateContainerPaths() map[string]string {
 		"pod-perceiver":         fmt.Sprintf("%s/%s/%s:%s", config.Registry, config.ImagePath, config.PodPerceiverImageName, config.PerceiverContainerVersion),
 		"image-perceiver":       fmt.Sprintf("%s/%s/%s:%s", config.Registry, config.ImagePath, config.ImagePerceiverImageName, config.PerceiverContainerVersion),
 		"perceptor-imagefacade": fmt.Sprintf("%s/%s/%s:%s", config.Registry, config.ImagePath, config.ImageFacadeImageName, config.ImageFacadeContainerVersion),
+		"perceptor-skyfire":     fmt.Sprintf("gcr.io/gke-verification/blackducksoftware/%s:%s", config.SkyfireImageName, config.SkyfireContainerVersion),
```
[the remark this answers is not in the document]
Comment (msenmurugan) [reply pkg/protoform/installer.go]: Since the Skyfire image doesn't push to blackducksoftware Docker hub, used the hard coded registry and image path, so that it can be used in downstream project

File: pkg/protoform/installer_test.go [hunk 2]
```diff
@@ -33,8 +33,9 @@ func TestProto(t *testing.T) {
 	d.DefaultCPU = "300m"
 	d.DefaultMem = "1300Mi"
 
-	installer := NewInstaller(d, "/etc/protoform")
+	installer := NewInstaller(d, "../../cmd/protoform.json")
 	installer.init()
+	installer.deploy()
```
[the remark this answers is not in the document]
Comment (msenmurugan) [reply pkg/protoform/installer_test.go]: Since the installer_test will run in Dryrun mode as "true", it will not deploy the containers

File: cmd/protoform.json [hunk 3]
```diff
@@ -4,5 +4,5 @@
   "DefaultCPU": "300m",
   "DefaultMem": "1300Mi",
   "InternalDockerRegistries": ["docker-registry.default.svc:5000", "172.1.1.0:5000"],
-  "PerceptorSkyfire": "true"
+  "PerceptorSkyfire": "on"
```
[the remark this answers is not in the document]
Comment (msenmurugan) [reply cmd/protoform.json]: Fixed it.

File: cmd/protoform.go [hunk 4]
```diff
@@ -41,7 +46,7 @@ func createDefaults() *api.ProtoformDefaults {
 	d.DefaultVersion = "master"
 	d.Registry = "gcr.io"
 	d.ImagePath = "gke-verification/blackducksoftware"
-	d.Namespace = "perceptor"
+	d.Namespace = "bds-perceptor"
```
[the remark this answers is not in the document]
Comment (msenmurugan) [reply cmd/protoform.go]: Modified it.

File: pkg/protoform/config.go [hunk 5]
```diff
@@ -159,14 +164,15 @@ func generateStringFromStringArr(strArr []string) string {
...
 	}
 
 	if p.PerceptorSkyfire {
-		configs["skyfire"] = map[string]string{"skyfire_conf.yaml": fmt.Sprint(`{"UseInClusterConfig": "`, "true", `","Port": "`, "3005", `","HubHost": "`, p.HubHost, `","HubPort": "`, p.HubPort, `","HubUser": "`, p.HubUser, `","HubUserPasswordEnvVar": "`, p.HubUserPasswordEnvVar, `","HubClientTimeoutSeconds": "`, p.HubClientTimeoutScannerSeconds, `","PerceptorHost": "`, p.PerceptorImageName, `","PerceptorPort": "`, p.PerceptorPort, `","ImageFacadePort": "`, p.ImageFacadePort, `","LogLevel": "`, p.LogLevel, `"}`)}
+		configs["skyfire"] = map[string]string{"skyfire.yaml": fmt.Sprint(`{"UseInClusterConfig": "`, "true", `","Port": "`, "3005", `","HubHost": "`, p.HubHost, `","HubPort": "`, p.HubPort, `","HubUser": "`, p.HubUser, `","HubUserPasswordEnvVar": "`, p.HubUserPasswordEnvVar, `","HubClientTimeoutSeconds": "`, p.HubClientTimeoutScannerSeconds, `","PerceptorHost": "`, p.PerceptorImageName, `","PerceptorPort": "`, p.PerceptorPort,
+			`","KubeDumpIntervalSeconds": "`, "15", `","PerceptorDumpIntervalSeconds": "`, "15", `","HubDumpPauseSeconds": "`, "30", `","ImageFacadePort": "`, p.ImageFacadePort, `","LogLevel": "`, p.LogLevel, `"}`)}
```
Comment (rrati) [inline pkg/protoform/config.go]: Why did you add this extra newline?

File: pkg/protoform/installer.go [hunk 6]
```diff
@@ -103,7 +120,7 @@ func (i *Installer) readConfig(configPath string) {
 		panic("No hub database password secret supplied.  Please inject PCP_HUBUSERPASSWORD as a secret and restart")
 	}
 
-	viper.AddConfigPath(configPath)
+	// viper.AddConfigPath(configPath)
```
Comment (rrati) [inline pkg/protoform/installer.go]: Remove this comment if code isn't needed anymore

File: pkg/protoform/installer_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (rrati) [reply pkg/protoform/installer_test.go]: Then why is this call needed?  init should be creating the resources and all deploy is doing is actually submitting them to the cluster.

File: cmd/protoform.json [hunk 7]
```diff
@@ -4,5 +4,16 @@
...
+  "HubHost": "nginx-webapp-logstash",
+  "HubPort": "8443",
+  "DefaultVersion": "master",
+  "Registry": "gcr.io",
+  "ImagePath": "gke-verification/blackducksoftware",
+  "Namespace": "perceptor",
+  "LogLevel": "info",
+  "HubClientTimeoutPerceptorSeconds": 5,
+  "HubClientTimeoutScannerSeconds": 120,
+  "ConcurrentScanLimit": 7
```
Comment (rrati) [inline cmd/protoform.json]: These changes don't seem necessary.  It doesn't seem like protoform.json is used anywhere.  We should probably remove it.
